### ISSUE TITLE
[fuchsia] a11y tests failing on topaz under ASAN

### ISSUE THREAD

**Issue by iskakaushik** [flutter/flutter#37820]
mirror for FL-291

**Comment by dnfield** [comment]
Pushing this out I have some leave coming up and am unlikely to get through this before I go on it (prioritizing getting more menaingful tests running on flutter_runner) - I've filed a separate bug for generally testing the engine using ASAN here, which I think would also help this:

**Comment by dnfield** [comment]
https://github.com/flutter/flutter/issues/40023

**Comment by dnfield** [comment]
@chinmaygarde - do the tests we run on LUCI use ASAN?  Is fixing this just a matter of adding some GN flag?

**Comment by dnfield** [comment]
I believe this is fixed - I'm going to close it and will re-open if we're finding that asan is failing again.
